src/components/application_manager/test/commands/mobile/system_request_test.cc

```diff
@@ -43,6 +43,10 @@
 #include "application_manager/event_engine/event.h"
 #include "application_manager/mock_hmi_interface.h"
 #include "application_manager/policies/mock_policy_handler_interface.h"
+#include "policy/policy_regular/policy/mock_policy_settings.h"
+#include "policy/policy_regular/policy/mock_policy_manager.h"
```

Comment:
@BSolonenko please be careful when include policy specific flow to the common tests. Wrap this includes and all related classes into `#ifdef PROPRIETARY_MODE` because there would be a problems with EXTERNAL_PROPRIETARY_MODE

Reply:
@AKalinich-Luxoft Fixed in https://github.com/smartdevicelink/sdl_core/pull/2151/commits/76d3c0c982069e44744428395178dd8c9f7412e4